microservices-connector/test/e2e/e2e_test.go

```diff
@@ -30,7 +30,7 @@ var _ = Describe("controller", Ordered, func() {
 		_, _ = utils.Run(cmd)
 	})
 
-	AfterAll(func() {
+	after all(func() {
```

Comment:
There is a bug imported by `[pre-commit.ci] auto fixes from pre-commit.com hooks`